packages/terra-modal-manager/themeable-variables.json

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
Hmm, this file shouldn't be generated here. Might talk with @emilyrohrbough and see if this is a side-effect of the terra-site updates. Might need to update the themeable-variables plugin to account for the new directory structure.

Reply:
If you delete this file, is it still being generated when you run terra-site.